test/unittest/statistics/dds/CMakeLists.txt

```diff
@@ -236,6 +236,7 @@ if (SQLITE3_SUPPORT AND FASTDDS_STATISTICS)
         ${PROJECT_SOURCE_DIR}/src/cpp/rtps/xmlparser/XMLParser.cpp
         ${PROJECT_SOURCE_DIR}/src/cpp/rtps/xmlparser/XMLParserCommon.cpp
         ${PROJECT_SOURCE_DIR}/src/cpp/rtps/xmlparser/XMLProfileManager.cpp
+        ${PROJECT_SOURCE_DIR}/src/cpp/security/artifact_providers/FileProvider.cpp
```

Comment:
This source file should be included in case `SECURITY` is enabled (from line 277 on)

Reply:
That's absolutely right!